File: SampleSetup/Output/Swift/Body/ApiBodyWithNested.swift
struct ApiBodyWithNested: Codable {
    var Prop1: String = ""
    var Prop2: Int? = nil

    struct Nested_1: Codable {
        var Prop_1_1: String = ""
        var Prop_1_2: Int? = nil

        struct Nested_1_2: Codable {
            var Prop_1_2_1: String = ""
            var Prop_1_2_2: Int? = nil
        }
    }
}

File: SampleSetup/Output/PHP_8/Body/ApiBodyWithNested.php
<?php
namespace SampleProject\Protocol\Body;

class ApiBodyWithNested {
    public string $Prop1;
    public ?int $Prop2;

    class Nested_1 {
        public string $Prop_1_1;
        public ?int $Prop_1_2;

        class Nested_1_2 {
            public string $Prop_1_2_1;
            public ?int $Prop_1_2_2;

    public function __construct()
    {
        $this->Prop_1_2_1 = "";
        $this->Prop_1_2_2 = null;
    }
        }

    public function __construct()
    {
        $this->Prop_1_1 = "";
        $this->Prop_1_2 = null;
    }
    }

    public function __construct()
    {
        $this->Prop1 = "";
        $this->Prop2 = null;
    }
}
